modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/Dump.java

```diff
@@ -226,13 +228,21 @@ public List<StoredCacheData> configs(String node, int group) {
      */
     public List<Integer> partitions(String node, int group) {
         File[] parts = dumpGroupDirectory(node, group)
-            .listFiles(f -> f.getName().startsWith(PART_FILE_PREFIX) && f.getName().endsWith(DUMP_FILE_EXT));
+            .listFiles(f -> f.getName().startsWith(PART_FILE_PREFIX) &&
```

Comment:
We have acceess to dump meta here.
So we know exactly what file extension we are expecting to find.